lib/resources/pip.rb

```diff
@@ -75,7 +81,7 @@ def pip_cmd
           return nil
         end
       end
-      pipcmd || 'pip'
+      pipcmd = @virtualenv_path.nil? ? 'pip' : @virtualenv_path
```

Comment:
@srenatus I updated it. let me know if the new one makes more sense? I'm new to this so feel free to [nit]